src/db/migrations/31-add-relics-season-1.ts

```diff
@@ -0,0 +1,48 @@
+import { Knex } from 'knex';
+
+import { AudioUrlExtractorTypes, IdExtractorTypes, TitleExtractorTypes } from '../../types/fieldExtractor';
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { addNftFactory, addPlatform, removeNftFactory, removePlatform } from '../migration-helpers';
+
+const RELICS_YXZ: MusicPlatform = {
+  id: 'relics',
+  type: MusicPlatformType['multi-track-multiprint-contract'],
+  name: 'relics',
+}
+
+const SEASONS_1: NftFactory = {
+  id: '0x441C1266E6fb13C38c2752eab0D11A99905FFef4',
+  startingBlock: '14180427',
+  platformId: 'relics',
+  contractType: NFTContractTypeName.default,
+  standard: NFTStandard.ERC721,
+  autoApprove: true,
+  approved: true,
+  typeMetadata: {
+    overrides: {
+      artist: {
+        // TODO: add extractor for artist ID?
+        name: 'RELICS',
+        websiteUrl: 'https://relics.xyz',
+        avatarUrl: 'https://web3-music-pipeline.mypinata.cloud/ipfs/QmS6WbpHrMYsu6Tmts61FfqiEwmrzJRrPrnN9o5qGDzME4',
```

Comment:
this link doesn't open an image properly - do you know what filetype this actually is?

Reply:
It's a .avif file, I'll do a quick conversion and replace it with .jpg

Reply:
I've just tried it with local spinamp instance and the image displays. I think we should leave it seeing as it works in the browser, or not?